src/tests/stub_cache_manager.cc

```diff
@@ -17,7 +17,7 @@
 Mgr::Action::Pointer CacheManager::createNamedAction(char const* action) STUB_RETVAL(NULL)
 void CacheManager::Start(const Comm::ConnectionPointer &conn, HttpRequest * request, StoreEntry * entry)
 {
-    std::cerr << HERE << "\n";
+    std::cerr << "\n";
```

Comment:
This PR automatically removes HERE. Nothing else should be required.

AFAICT, the PR change does not have runtime effects in this case -- `HERE` outputs nothing. Please correct me if I am wrong.

As for how this line should be fixed, I am not sure, Maybe it should have `MYNAME`. Or maybe it should be removed. That is a question for a different PR.